src/managers/GuildManager.js

```diff
@@ -128,23 +128,29 @@ class GuildManager extends BaseManager {
     return super.resolveID(guild);
   }
 
+  /**
+   * Options used to create a guild.
+   * @typedef {Object} GuildCreateOptions
+   * @property {number} [afkChannelID] The ID of the AFK channel
```

Comment:
This can be a Snowflake too

Reply:
hmm, now that you pointed it out, I don't get why it is a number in the first place. Should be changed to `Snowflake` in typings too I think.